apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessment/assessment.html.ts

```diff
@@ -120,6 +125,18 @@ function AssessmentQuestionRow({
             : `${question.number_in_alternative_group}.`}
           ${question.title}
         </a>
+        ${question.manual_rubric_id == null
+          ? ''
+          : html`
+              <span
```

Comment:
See https://getbootstrap.com/docs/5.3/components/tooltips/#markup - you probably want an `<a href="#" ...>` to ensure that this is accessible to keyboard users. This is what we use in PT for explanatory popovers.

Reply:
Would this show an underline in BS5? Is that an issue?

Reply:
It shouldn't, at least, it doesn't over in PT.